README.md

```diff
@@ -47,10 +53,15 @@ but all the dependency modules will remain in the system.
 
 ## Project contents
 
-`__main__.py` is project executable. It uses the following modules from `universum` directory:
-* `main`/`poll`/`submit`/`api` - managing modules for different Universum modes
+`universum` is main project folder, that is being copied to `dist_packages` when installed.
+It contains `__main__.py` script, that is the main entry point to the whole project.
+It also contains the following modules:
+* `main`/`poll`/`submit`/`api`/`nonci` - managing modules for different Universum modes
 * `configuration_support` - special module for [configuring the project](
 https://universum.readthedocs.io/en/latest/configuring.html)
+* `analyzers` directory is not quite a part of Universum itself. It contains [independent scripts](
+https://universum.readthedocs.io/en/latest/code_report.html) compatible with Universum
+for static (and other types of) analysis.
```

Comment:
I think it should be changed a little (not related to python3.7): "for static" -> "for implementing static"
This is because otherwise it looks like these script implement static analysis themselves.
Maybe you can also mention that these scripts wrap 3rd-party tools.